train-habitat.gemspec

```diff
@@ -14,9 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage    = "https://github.com/inspec/train-habitat"
   spec.license     = "Apache-2.0"
 
-  spec.files = %w{
-    train-habitat.gemspec Gemfile
```

Comment:
FYI, for train plugins, skipping the gemspec is safe, but for inspec plugins we are (or at least we were) needing the gemspec to be shipped.

Reply:
👍